packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -136,7 +136,10 @@ export function useSpecificEditorSettings() {
 				templateSlug: _record.slug,
 				canvasMode: getCanvasMode(),
 				settings: getSettings(),
-				postWithTemplate: _context?.postId,
+				postWithTemplate:
```

Comment:
"posts" actually have template, so I would change the value of this flag personally.

 - So either we should rename to something about `shouldUseTemplateAsDefaultRenderingMode` if it's only used for that config.
 - Move the "post type" check to later (when setting the `defaultRenderingMode` variable.

Reply:
Makes sense to rename and was only used for the rendering mode selection, so I renamed to `shouldUseTemplateAsDefaultRenderingMode.